### PR TITLE
fix: otrs module upgrade

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
Close MBE-83 & MBE-90

### Requirements

Unable to copy the ticket number from the OTRS pop-up after a ticket is created

### Description of the Change

This issue has been fixed after upgrade to the latest OTRS module, which uses draggable library.

**Link:** https://github.com/ovh-ux/ng-ovh-otrs/pull/38

### CODE REVIEW

**Comment by ganeshkumar1989** [comment]
Currently blocked, until https://github.com/ovh-ux/ng-ovh-otrs/pull/38 is released

File: client/app/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ import 'script-loader!@bower_components/international-phone-number/releases/inte
 import 'script-loader!@bower_components/qrcode/lib/qrcode.js';
 import 'angular-qr';
 import 'script-loader!u2f-api-polyfill/u2f-api-polyfill.js';
-import 'ovh-angular-otrs';
+import '@ovh-ux/ng-ovh-otrs';
```
Comment (antleblanc) [inline client/app/index.js]: Could you move this import statement directly to `client/app/app.js`?

```suggestion
import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
```

Comment (ganeshkumar1989) [reply client/app/index.js]: This is done :)

Comment (antleblanc) [reply client/app/index.js]: Thank you!

**Comment by JeremyDec** [comment]
@ganeshkumar1989 => conflicts :)

**Comment by ganeshkumar1989** [comment]
> @ganeshkumar1989 => conflicts :)

Thanks :)
Its if fixed now

**Comment by antleblanc** [comment]
@ganeshkumar1989 You can now benefit from the latest release of the OTRS module (ref: https://github.com/ovh-ux/ng-ovh-otrs/blob/master/CHANGELOG.md#700-2019-03-11)

**Comment by ganeshkumar1989** [comment]
> @ganeshkumar1989 You can now benefit from the latest release of the OTRS module (ref: https://github.com/ovh-ux/ng-ovh-otrs/blob/master/CHANGELOG.md#700-2019-03-11)

Thanks a lot @antleblanc . The latest release of the otrs module is now being used (updated in this PR)

**Pull request merged**